src/rust/rust-proto/src/client_macros.rs

```diff
@@ -32,10 +32,10 @@ macro_rules! execute_client_rpc {
         $opts: expr,
     ) => {{
         {
-            let backoff = client_executor::strategy::FibonacciBackoff::from_millis(100)
+            let backoff = client_executor::strategy::FibonacciBackoff::from_millis(10)
                 .max_delay(Duration::from_millis(5000))
                 .map(client_executor::strategy::jitter);
-            let num_retries = 10;
+            let num_retries = 1000;
```

Comment:
1000 retries feels wildly incorrect, especially when after fib(20)*10 of them each retry will delay up to 5sec long.

If this keeps retrying we'd just find outselves waiting for ~5sec * 980

Reply:
Agree, if an RPC is failing for even a matter of minutes I think we're beyond "retry" and we're in "a dependency is failing, we are no longer healthy".

Reply:
Yes, I agree as well. This was an attempt to eliminate this variable as a possible cause for the observed failure.